src/main/kotlin/org/rust/lang/core/psi/RsPsiFactory.kt

```diff
@@ -18,6 +18,15 @@ class RsPsiFactory(private val project: Project) {
         createFromText("fn main() { $text; }")
             ?: error("Failed to create expression from text: `$text`")
 
+    fun createTryExpression(expr: RsExpr): RsTryExpr {
+        val newElement = createExpression("a?")
+        if (newElement is RsTryExpr) {
+            newElement.expr.replace(expr)
+            return newElement
+        }
+        error("Failed to create try expression ${expr.text}?")
```

Comment:
Could you add backtics around `${expr.text}`, as is done in other cases in this file? 

This is useful, because it will nicely delimit trailing/leading whitespace, if there's any (Rust's `Debug` for strings which encloses the string in `"` and escapes `\n` is a better solution for this problem, but there isn't `Debug` in Java).